pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -41,9 +42,41 @@ import (
 )
 
 const (
-	defaultConfigHistoryLimit = 10
-
+	defaultConfigHistoryLimit     = 10
+	defaultCacheHeaterCronImage   = "bitnami/kubectl:latest"
+	defaultCacheHeaterSchedule    = "* * * * *"
 	defaultPortAllocationResource = "default"
+	volumeTeamLabel               = "tsuru.io/volume-name"
```

Comment:
The label is `volume-name` and the variable is `volumeTeam`, is this correct?

Reply:
No, it is our mistake, renaming to `tsuru.io/volume-team`